docs/extras/integrations/llms/promptguard.ipynb

```diff
@@ -0,0 +1,215 @@
+    "\n",
+    "os.environ['PROMPT_GUARD_ACCESS_TOKEN'] = \"<PROMPT_GUARD_API_KEY>\"\n",
+    "os.environ['OPENAI_API_KEY'] = \"<OPENAI_API_KEY>\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use PromptGuardLLMWrapper\n",
+    "\n",
+    "Applying promptguard to your application could be as simple as wrapping your LLM using the PromptGuardLLMWrapper class. \n",
+    "```python\n",
+    "# llm=OpenAI()\n",
+    "llm=PromptGuardLLMWrapper(OpenAI())\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import langchain\n",
+    "from langchain import LLMChain, PromptTemplate\n",
+    "from langchain.callbacks.stdout import StdOutCallbackHandler\n",
+    "from langchain.llms import OpenAI\n",
+    "from langchain.memory import ConversationBufferWindowMemory\n",
+    "\n",
+    "from langchain.llms import PromptGuardLLMWrapper\n",
+    "\n",
+    "langchain.verbose = True\n",
+    "langchain.debug = True\n",
+    "\n",
+    "prompt_template = \"\"\"\n",
+    "As an AI assisant, you will answer questions according to given context.\n",
+    "\n",
+    "Important PII data is sanitized in the question.\n",
+    "For example, \"Giana is good\" is sanitized to \"PERSON_999 is good\".\n",
+    "You must treat the sanitized data as opaque strings, but you can use them as\n",
+    "meaningful entities in the response.\n",
+    "Different sanitized items could be the same entity based on the semantics.\n",
+    "You must keep the sanitized item as is and cannot change it.\n",
+    "The format of sanitized item is \"TYPE_ID\".\n",
+    "You must not create new sanitized items following the format. For example, you\n",
+    "cannot create \"PERSON_1000\" or \"PERSON_998\" if \"PERSON_1000\" or \"PERSON_998\" is\n",
+    "not in the question.\n",
+    "\n",
+    "Conversation History: ```{history}```\n",
+    "Context : ```Mr. Carl Smith is a 31-year-old man who has been experiencing\n",
+    "homelessness on and off for all his adult life. Mr. Smith says he is about\n",
+    "5’5” and weighs approximately 129 lbs. He presents as\n",
+    "very thin, typically wearing a clean white undershirt\n",
+    "and loose-fitting khaki shorts at interviews.\n",
+    "His brown hair is disheveled and dirty looking, and\n",
+    "he constantly fidgets and shakes his hand or\n",
+    "knee during interviews. Despite his best efforts, Carl is a poor historian. ```\n",
+    "Question: ```{question}```\n",
+    "\n",
+    "\"\"\"\n",
+    "\n",
+    "chain = LLMChain(\n",
+    "    prompt=PromptTemplate.from_template(prompt_template),\n",
+    "    llm=PromptGuardLLMWrapper(llm=OpenAI()),\n",
+    "    memory=ConversationBufferWindowMemory(k=2),\n",
+    "    verbose=True,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "print(\n",
+    "    chain.run(\n",
+    "        {\"question\": \"\"\"How high is he? \"\"\"},\n",
```

Comment:
Is this intended to be "How tall is he"? Same comment everywhere else this shows up

Reply:
"How high is he?"
"At least 2 pounds!"